src/Avalonia.Themes.Fluent/Accents/FluentControlResources.xaml

```diff
@@ -207,6 +207,11 @@
       <StaticResource x:Key="MenuFlyoutPresenterBackground" ResourceKey="SystemControlTransientBackgroundBrush" />
       <StaticResource x:Key="MenuFlyoutPresenterBorderBrush" ResourceKey="SystemControlTransientBorderBrush" />
       <Thickness x:Key="MenuFlyoutPresenterBorderThemeThickness">1</Thickness>
+      <!-- Mobile Resources for MenuFlyout.xaml -->
+      <x:Double x:Key="MobileMenuFlyoutThemeMinWidth">32</x:Double>
+      <x:Double x:Key="MobileMenuFlyoutSeparatorThemeWidth">1</x:Double>
+      <Thickness x:Key="MobileMenuFlyoutItemMargin">2,0,2,0</Thickness>
+      <Thickness x:Key="MobileMenuFlyoutItemThemePaddingNarrow">8,4,8,7</Thickness>
```

Comment:
Maybe instead of "mobile" we should use "horizontal" naming for that.
Text selection handles and text selection flyouts aren't mobile only. It's essentially touch-specific, and there are a lot of touch devices on Windows/embed linux.